client/components/QuestionsAndAnswers/MasterQA.jsx

```diff
@@ -1,33 +1,33 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 import AddQuestion from './AddQuestion';
-import ModalForm from './ModalForm';
+// import { QuestionModal, AnswerModal } from './ModalForm';
+// import { QuestionModal, AnswerModal } from './ModalForm';
 import MoreQuestions from './MoreQuestions';
 import QuestionList from './QuestionList';
 import SearchQuestion from './SearchQuestion';
 import Data from './dummyData';
 
 // component
-const compareFn = (a, b) => b.question_helpfulness - a.question_helpfulness;
-
 const MasterQA = () => {
-  const sortedQuestions = Data.questions.results.sort(compareFn);
+  const sortedQuestions = Data.questions.results.sort(
```

Comment:
I believe Jess realized that sorting may be done on the backend/via the API vs needing to be done manually for her module, so the same may be applicable for your module. This doesn't need to be changed now, just something to note when you get to working with the API. Check in with her about sorting in case I'm remembering incorrectly.

Reply:
That's a good point, something to keep in mind for sure